services/storage/src/simcore_service_storage/utils/simcore_s3_dsm_utils.py

```diff
@@ -143,20 +145,38 @@ def create_random_export_name(user_id: UserID) -> StorageFileID:
     )
 
 
+def ensure_same_paret_in_user_selection(object_keys: list[S3ObjectKey]) -> bool:
+    parents = [Path(x).parent for x in object_keys]
+    return len(set(parents)) <= 1
+
+
+UserSelection: TypeAlias = str
+
+
+def _strip_parent(selection: UserSelection, s3_object: S3ObjectKey) -> str:
+    selection_path = Path(selection)
```

Comment:
you do not want to resolve them first ? i.e. Path(...).resolve()

Reply:
I see it's not related to what I'm trying to do. So Tests seem happy so I will not change it instead if there's an issue